Automattic/pocket-casts-android
# Bookmarks: Add empty/ loading/ upsell views

## Description
This adds bookmarks empty/ loading/ upsell views

## Testing Instructions

1. Install debug build without login or login with a free account
3. Go to `Discover`
4. Tap a podcast and play an episode
5. Open full player
6. ✅ Notice Plus upsell view matches `Figma` design (Io7PCz1H1hmOgEE9eljoYW-fi-2100%3A34078)
7. Dismiss the player 
8. Logout
9. Login with a Plus account
10. Repeat steps 3-5
11. ✅  Notice no bookmarks view matches `Figma` design (Io7PCz1H1hmOgEE9eljoYW-fi-2032%3A45279)
12. ✅  Notice that content is scrollable in landscape mode with large font


## Screenshots or Screencast 

Plus Upsell | No bookmarks | Large font portrait
-----|-----|---
<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/4e008928-2306-4998-9bfc-89ed628ea32f"/>| <img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/20465949-804e-48d2-8e83-c4635f79732c"/> | <img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/0a0b5fbc-03db-4fff-9c45-d10afa5be9a3"/>

|Large font landscape |
| ----|
<img width=500 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/062154f5-be50-4ee4-a52a-29f45e03b707"/>

## Checklist
 
#### I have tested any UI changes...
